curriculum/challenges/english/01-responsive-web-design/css-piano/part-003.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 612e78af05201622d4bab8aa
+title: Part 3
+challengeType: 0
+dashedName: part-3
+---
+
+# --description--
+
+Time to start working on the piano. Within your `body` element, create a `div` element with the `id` set to `piano`.
```

Comment:
```suggestion
Time to start working on the piano. Create a ' div ' element within your `body` element with the `id` set to `piano`.
```

Reply:
Maybe a bit nitpicky but I like this wording more.

Reply:
Why the spacing and use of single quotes?

Reply:
my bad